source/host.c

```diff
@@ -282,7 +286,9 @@ void Host_WriteConfiguration (void)
 		}
 
 		Key_WriteBindings (f);
+#ifndef __WII__
```

Comment:
We should support dt binds on all platforms even if we don't personally plan to ship them by default.